drivers/include/at24mac.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2019 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_at24mac unique ID chip
+ * @brief       Device driver interface for the AT24MAC I2C chip
+ * @{
+ *
+ * @file
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ */
+
+#ifndef AT24MAC_H
+#define AT24MAC_H
+
+#include <stdint.h>
+#include "net/eui48.h"
+#include "net/eui64.h"
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Device handle type for AT24Mac devices
+ */
+typedef uint_fast8_t at24mac_t;
+
+#define AT24MAC_ID_LEN  (16)            /**< Length of ID128 */
+
+/**
+ * @brief   Type of the AT24Mac device
+ */
+typedef enum __attribute__ ((__packed__)) {
```

Comment:
Why is `__attribute__ ((__packed__))` required ?

Reply:
Since we use `-fshort-enums`, we don't need this.